docker-compose.yml

```diff
@@ -48,7 +48,7 @@ services:
       - ${WEB_PORT}:${WEB_PORT}
       - ${WEB_EXPOSED_PORT}:${WEB_EXPOSED_PORT}
     volumes:
-      - ${CERT_PATH}:/etc/ssl/certs
+      - ${CERT_PATH}:/etc/ssl
```

Comment:
I didnt see anything about this in the Readme. Have you tried changing this path? What happens if it is not a valid dir in the container? Should we clearly say that "it has to be a valid path in the container" or something around the lines of what that path means?

Reply:
I have added through comments for all the variable on the .env sample file and mentioned in the read me file to read those comments while entering env var values